test/boost/memtable_test.cc

```diff
@@ -121,12 +121,20 @@ SEASTAR_TEST_CASE(test_memtable_with_many_versions_conforms_to_mutation_source)
             }
 
             return mt->as_data_source();
-        });
+        }, true);
```

Comment:
Why explicit `true`? It's defaulted by test/lib/mutation_source_test.hh the same way run_mutation_source_tests does

Reply:
Because AFAIK the signature of `run_tests` in `test_memtable()` can't have a default argument.

*Maybe* there could be some lambda hack but it would make the code even less readable.

Reply:
Ah, indeed

Reply:
Can you please approve this? thanks